tower-balance/src/pool.rs

```diff
@@ -284,13 +290,19 @@ where
 
             let discover = self.balance.discover_mut();
             if self.ewma < self.options.low {
+                if discover.load != Level::Low {
+                    tracing::trace!({ ewma = %self.ewma }, "pool is over-provisioned");
```

Comment:
TIOLI: what if all of these conditionally-recorded traces were replaced with a single
```rust
tracing::trace!(ewma = %self.ewma, %discover.load, %discover.services)
```
or something?

Reply:
That would result in a _lot_ of mostly unnecessary traces (the ewma is changing for _every_ request, whereas the load estimate changes only very rarely). Not sure about this one.

Reply:
Ah, yeah, you're right. Never mind, let's not do that one.